software/src/modules/ac011k/ac011k.cpp

```diff
@@ -1493,8 +1446,37 @@ void AC011K::loop()
                 }
                 break;
 
+            case 0x0C:
+                /*
+                              0  1  2  3  4  5  6  7  8  9  0  10 11 12 13 14 
+                         000: FA 03 00 00 0C 02 05 00 11 09 01 00 00 73 79            CtrlGetS20OpenStop -> Off
+                         000: FA 03 00 00 0C 03 05 00 11 0A 01 00 00 B2 F1            CtrlGetS20OPENLOCK -> Off
+                         000: FA 03 00 00 0C 01 05 00 11 0B 01 00 00 32 D4            CtrlGetRemoteStart -> Off
+                         000: FA 03 00 00 0C 04 05 00 11 0C 01 00 00 F3 9F            CtrlGetOfflineStop -> Off
+                         000: FA 03 00 00 0C 06 08 00 11 0D 04 00 B8 0B 00 00 46 76   CtrlGetOfflineEnergy -> 0x0bb8 = 3000Wh
+                 */ 
+                switch( PrivCommRxBuffer[9] ) {
+                    case 0x09:
+                        logger.printfln("CtrlGetS20OpenStop = %d", PrivCommRxBuffer[12]);
+                        break;
+                    case 0x0A:
+                        logger.printfln("CtrlGetS20OPENLOCK = %d", PrivCommRxBuffer[12]);
+                        break;
+                    case 0x0B:
+                        logger.printfln("CtrlGetRemoteStart = %d", PrivCommRxBuffer[12]);
+                        break;
+                    case 0x0C:
+                        logger.printfln("CtrlGetOfflineStop = %d", PrivCommRxBuffer[12]);
+                        break;
+                    case 0x0D:
+                        logger.printfln("CtrlGetOfflineEnergy %dWh", PrivCommRxBuffer[12]+256*PrivCommRxBuffer[13]);
```

Comment:
please see length field in bytes 10 and 11, the value following has 4 bytes length, this is an unsigned long value, not an unsigned short